modules/core/src/main/java/org/apache/ignite/cache/QueryEntity.java

```diff
@@ -669,6 +687,74 @@ public QueryEntity addQueryField(String fullName, String type, String alias) {
         return this;
     }
 
+    /**
+     * Returns INLINE_SIZE for PK index.
+     *
+     * @return INLINE_SIZE for PK index.
+     */
+    public int getPrimaryKeyInlineSize() {
+        return pkInlineSize != null ? pkInlineSize : -1;
+    }
+
+    /**
+     * Sets INLINE_SIZE for PK index. Implemented at the child.
```

Comment:
Which implementation in the child do you mean here?

Reply:
It's a mistake. Thanks.